algrtm/src/components/OpportunitiesIndustrySection.js

```diff
@@ -0,0 +1,36 @@
+"use client";
+import React from "react";
+import CollapsibleSectionsComponent from "./shared/CollapsibleSectionsComponent";
+import Image from "next/image";
+
+const OpportunitiesIndustrySection = () => {
+  return (
+    <section className="flex flex-col w-full my-[4rem] lg:my-[7.19rem] gap-[2.75rem] justify-center items-center text-center">
+      <h2 className="flex h3SB xl:h2 text-neutralLight-neutral10 dark:text-neutralDark-neutral10 tracking-[-0.13rem]lg:tracking-[-0.13rem] xl:tracking-[-0.16rem]">
+        Opportunities in Data Science <br />
+        for Heavy Industry
+      </h2>
+      <div className="flex flex-col gap-4 w-full xl:px-[5.875rem]">
+        <div className="flex flex-col md:flex-row gap-[1.5rem]  bg-neutralLight-neutral90 dark:bg-neutralDark-neutral90 px-[2.75rem] py-[2.75rem] md:py-[5.88rem]  xl:py-[5.85rem] xl:px-[6.75rem] rounded-[2rem]">
+          <div className=" h5_small_bold md:h4SB lg:h4 text-neutralLight-neutral10 dark:text-neutralDark-neutral10 flex justify-start max-w-[25.1875rem] w-full">
+            <Image
+              src="/factory-image.png"
+              alt="Image of the factory inside"
+              width={440}
+              height={554}
+              sizes="100vw"
+              style={{
+                width: "100%",
+                height: "auto",
+              }}
+            />
+          </div>
+
+          <CollapsibleSectionsComponent version="version3" />
+        </div>
```

Comment:
Something weird is happening here with the image. When we reveal the text, the image stretches:

![image](https://github.com/Nospoko/algrtm/assets/26410548/6efa9f68-a320-4c91-b116-b285dc5dc589)

On smaller devices, the text should not be centered, and there is something wrong with the icons:

![image](https://github.com/Nospoko/algrtm/assets/26410548/64c6155f-258a-4e45-8c59-f7d1a3f38dea)